arduino-ide-extension/src/test/node/arduino-daemon-impl.test.ts

```diff
@@ -114,18 +114,22 @@ describe('arduino-daemon-impl', () => {
   // });
 
   it('should parse the port address when the log format is json', async () => {
-    const instance = await new SilentArduinoDaemonImpl(
+    const { daemon, port } = await new SilentArduinoDaemonImpl(
       'json'
     ).spawnDaemonProcess();
-    expect(instance.port).not.to.be.undefined;
-    expect(instance.port).not.to.be.equal('0');
+
+    expect(port).not.to.be.undefined;
+    expect(port).not.to.be.equal('0');
+    daemon.kill();
```

Comment:
Maybe I'd prefer to put the `new SilentArduinoDaemonImpl('json').spawnDaemonProcess();` in the `beforeEach` and the `daemon.kill()` in the `afterEach`.
@fstasi does it make sense to you?

Reply:
we actually run the `SilentArduinoDaemonImpl` with different input: `json` or `text`.. we can make a wrapper function probably